### DISCUSSION TITLE
mosdns支持api吗

### DISCUSSION THREAD

**Discussion by Cause-jun** [kkkgo/blog.03k.org#29] [Q&A]
我按照文档上写了api的http配置信息配置之后报错Error: fail to load config, failed to unmarshal config: 1 error(s) decoding:

* '' has invalid keys: api
2024-05-23T10:10:06.174+0800	FATAL	fail to load config, failed to unmarshal config: 1 error(s) decoding:

* '' has invalid keys: api

**Comment by kkkgo** [comment]
如果你是指PaoPaoDNS的mosdns的话，那个是魔改的，并不是原来的mosdns
